src/App.tsx

```diff
@@ -1,61 +1,100 @@
 import './App.scss';
+import { useState } from 'react';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+
+import { TodoList } from './components/TodoList';
+
+import { todosWithUsers } from './utils/getTodoesWithUsers';
+import { getRandomId } from './utils/getRandomId';
+import { findUserById } from './utils/findUserById';
 
 export const App = () => {
+  const [userId, setUserId] = useState(0);
```

Comment:
Here you can use a variable like this: ``const DEFAULT_USER_ID = 0;`` to avoid magic values.